rfhub2/cli/rfhub_importer.py

```diff
@@ -39,34 +40,33 @@ def add_collections(self) -> None:
         for path in self.paths:
             self._traverse_paths(path)
         if not self.no_installed_keywords:
-            libdir = os.path.dirname(robot.libraries.__file__)
-            for item in os.listdir(libdir):
+            libdir = Path(robot.libraries.__file__).parent
+            for item in Path.iterdir(libdir):
                 if not self._should_ignore(item):
-                    self.add(os.path.join(libdir, item))
+                    self.add(item)
 
-    def _traverse_paths(self, path: str) -> None:
+    def _traverse_paths(self, path: Path) -> None:
         """
         Traverses through paths and adds libraries to rfhub.
         Helper function for add_collections.
         """
-        for item in os.listdir(path):
-            full_path = os.path.join(path, item)
-            if os.path.isdir(full_path):
-                if self._is_library_with_init(full_path):
-                    self.add(full_path)
+        for item in Path.iterdir(Path(path)):
```

Comment:
path and item are already instances of Path, can be just path.iterdir(), item.is_dir(), item.is_file() etc

Reply:
Changed. Turned out that path was passed as string instead of Path object. Now it's cast as Path.